pkg/totalaccountwatcher/totalaccountwatcher_test.go

```diff
@@ -114,7 +114,6 @@ func TestTotalAwsAccounts(t *testing.T) {
 
 			// Act
 			TotalAccountWatcher = NewTotalAccountWatcher(mocks.fakeKubeClient, mocks.mockAWSClient, 10)
-			TotalAccountWatcher.AwsClient = mocks.mockAWSClient
```

Comment:
Unrelated change, but this was redundant since `NewTotalAccountWatcher` does exactly this via the second arg.